src/main/java/halberstam/weather/CurrentWeatherFrame.java

```diff
@@ -0,0 +1,59 @@
+package halberstam.weather;
+
+import io.reactivex.rxjava3.disposables.Disposable;
+import io.reactivex.rxjava3.schedulers.Schedulers;
+import retrofit2.Retrofit;
+import retrofit2.adapter.rxjava3.RxJava3CallAdapterFactory;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class CurrentWeatherFrame extends JFrame {
+    public CurrentWeatherFrame()
+    {
+        setSize(650, 600);
+        setTitle("Current Weather");
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
+
+        JPanel enterCityPanel = new JPanel();
+        enterCityPanel.setLayout(new FlowLayout());
+
+        JTextField enterCityName  = new JTextField("Passaic             ");
```

Comment:
Can you make it so it automatically requests Passaic when the application starts up?

Reply:
is it better now?